vunit/vhdl/verification_components/run.py

```diff
@@ -86,13 +87,17 @@ def gen_avalon_master_tests(obj, *args):
 
 for test in TB_WISHBONE_SLAVE.get_tests():
     #  TODO strobe_prob not implemented in slave tb
-    gen_wb_tests(test, [8, 32], [1, 64], [1.0], [0.3, 1.0], [0.4, 0.0])
+    gen_wb_tests(test, [8, 32], [1, 64], [1.0], [0.3, 1.0], [0.4, 0.0], [True, ])
 
 
 TB_WISHBONE_MASTER = LIB.test_bench("tb_wishbone_master")
 
 for test in TB_WISHBONE_MASTER.get_tests():
-    gen_wb_tests(test, [8, 32], [1, 64], [0.3, 1.0], [0.3, 1.0], [0.4, 0.0])
+    if test.name == "slave comb ack":
```

Comment:
I'm not sure to understand this. Why is the combinatorial version tested with one set of params but the other one is tested with two?

Reply:
slave comb ack test case doesn't need/cannot have random stimulus